your-rents-data/src/test/java/com/yourrents/data/BaseDataTest.java

```diff
@@ -0,0 +1,77 @@
+package com.yourrents.data;
+
+/*-
+ * #%L
+ * Your Rents Data
+ * %%
+ * Copyright (C) 2019 - 2021 Your Rents Team
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
my personal taste, I'd prefer to use assertJ instead

Reply:
We can use both, or more. My preference is for Hamcrest.